src/components/UserMessageSection.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 export function UserMessageSection() {
   return (
-    <div id='user-message-section'>
+    <div className='user-message-section'>
       <p style={{ color: '#fff', fontSize: '200%' }}>
```

Comment:
I guess the following inline CSS will be removed once the design is finalized? :thinking:

Reply:
Yes. It's placeholder. I am currently working on it.